### ISSUE TITLE
The result of cross-lingual VC was bad

### ISSUE THREAD

**Issue by zzxiang** [espnet/espnet#3654]
I tried [egs/vcc20/tts1_en_zh](https://github.com/espnet/espnet/tree/master/egs/vcc20/tts1_en_zh) and [egs/vcc20/vc1_task2](https://github.com/espnet/espnet/tree/master/egs/vcc20/vc1_task2), and the result was bad. Even if I tried the downloaded finetuned TTS model, the result was still the same.

Did the same thing happen during the event of Voice Conversion Challenge 2020?

Here's the result of converting speech E30002 of source English speaker SEF1 to the voice of target Mandarin speaker TMF1:

https://user-images.githubusercontent.com/10098144/136896159-81720740-af08-4aaa-b008-67944fd8c142.mp4

The result was picked up from the folder of `egs/vcc20/vc1_task2/exp/SEF1_eval_asr/TMF1_train_pytorch_train_pytorch_transformer+spkemb.tts1_en_zh_model.loss.best_denorm/SEF1_TMF1_eval/pwg_wav`
 after following the recipe of `tts1_en_zh` and `vc1_task2`.

Here's the speech E30002 of source English speaker SEF1:

https://user-images.githubusercontent.com/10098144/136896528-1f6c30e2-0bc9-41af-99be-629916f8dc4b.mp4

And here's one speech of the training data of target Mandarin speaker TMF1:

https://user-images.githubusercontent.com/10098144/136897947-5886f9ed-205b-4c4e-9c84-cefb9848b466.mp4

I checked the ASR result of SEF1 speech (`egs/vcc20/vc1_task2/exp/SEF1_eval_asr/result/data.1.json`). Seems that the text of E00002 was correctly recognized:

```
        "SEF1_E30002": {
            "output": [
                {
                    "name": "target1[1]",
                    "rec_text": "▁KIOTO▁MUST▁NOT▁REMAIN▁AN▁EMPTY▁PROMISE<eos>",
                    "rec_token": "▁K IO TO ▁MUST ▁NOT ▁REMAIN ▁AN ▁EMPTY ▁PROMISE <eos>",
                    "rec_tokenid": "2662 207 424 3076 3158 3744 624 1784 3561 5001",
                    "score": -33.93509650230408,
                    "shape": [
                        1,
                        3
                    ],
                    "text": ".",
                    "token": ".",
                    "tokenid": "1"
                }
            ],
            "utt2spk": "SEF1"
        },
```

So I guess the reason lies in the pretrained or finetuned TTS model. I completely followed the vcc20 recipe without changing any parameter, except that when training my own model, I had to specify `--pretrained_model_dir` to `egs/vcc20/`.
